packages/SwingSet/src/kernel/kernel.js

```diff
@@ -34,10 +34,7 @@ import { notifyTermination } from './notifyTermination.js';
 import { makeVatAdminHooks } from './vat-admin-hooks.js';
 
 /**
- * @typedef {import('@agoric/swingset-liveslots').VatDeliveryObject} VatDeliveryObject
- * @typedef {import('@agoric/swingset-liveslots').VatDeliveryResult} VatDeliveryResult
- * @typedef {import('@agoric/swingset-liveslots').VatSyscallObject} VatSyscallObject
- * @typedef {import('@agoric/swingset-liveslots').VatSyscallResult} VatSyscallResult
+ * @import { VatDeliveryObject, VatDeliveryResult, VatSyscallObject, VatSyscallResult } from '@agoric/swingset-liveslots'
```

Comment:
:clap:

Reply:
I'm curious... can we do qualified imports? i.e. use `swingset.VatSyscallResult`?

Reply:
Not sure I understand the question. We can still use paths to pick modules out of the package exports.

Reply:
by qualified import, I mean, for example:

```js
/** @import * as swingset from '@agoric/swingset-liveslots' */
...
/**
 * @param {swingset.VatDeliveryResult} result
 */
```

Reply:
oh yes, it does. I've updated the example to work like in your comment